### PR TITLE
Refactor upgrade/install scripts to use FTP Class in Class-Packages.php

### PR DESCRIPTION
**Pull request by equinoxmatt**

### CODE REVIEW

File: Sources/Class-Package.php [hunk 1]
```diff
@@ -793,13 +793,15 @@ public function connect($ftp_server, $ftp_port = 21, $ftp_user = 'anonymous', $f
 		if (!$this->connection)
 		{
 			$this->error = 'bad_server';
+            $this->last_message = 'Invalid Server';
```
Comment (MissAllSunday) [inline Sources/Class-Package.php]: Can you change your spaces for tabs? SMF uses 4 spaces tabs, thanks!

Comment (equinoxmatt) [reply Sources/Class-Package.php]: I really wish I knew what was wrong here.  It looks fine when I pull down a fresh copy of the repo. Seems to be some weird line ending stuff going on!

Is this github making it look wrong?  I have both linux and windows machines,  at work and at home.  Wherever I view the file,  the formatting is correct.

Comment (MissAllSunday) [reply Sources/Class-Package.php]: Perhaps your editor is automagically changing tabs for spaces or viceversa when saving?

**Comment by equinoxmatt** [comment]
I fixed the line endings issue.  Thank god for Vim, was the only editor showing the line spacing issue.

**Comment by MissAllSunday** [comment]
Thanks!

**Pull request merged**